application-flashmessages-ui/src/main/resources/Flash/FlashNotifier.xml

```diff
@@ -42,9 +42,11 @@
   <content>{{velocity}}
 ## Mark the message as seen by the current user
 #if("$!request.ajax" == '1' &amp;&amp; $xcontext.action == 'get')
+  #set($dateNow = $datetool.getSystemDate())
```

Comment:
I've never used this. Does it have any advantage over ``$datetool.date``?

Reply:
There is no difference
Found it on https://velocity.apache.org/tools/devel/apidocs/org/apache/velocity/tools/generic/DateTool.html#getSystemDate--

Reply:
There is no difference 
Found it on https://velocity.apache.org/tools/devel/apidocs/org/apache/velocity/tools/generic/DateTool.html#getSystemDate--
And the same was used here https://github.com/xwikisas/application-flashmessages/pull/48/files/7a70b8080d68cd96b0d36d8a826a95e799ade180#diff-061b98730a9b5efba765e5bb5f8cb3a3L88
Is `$datetool.date` preferred to be used?

Reply:
I prefer the shorter version :) but I don't think there any best practice documented about this.